src/core.jl

```diff
@@ -108,6 +109,17 @@ struct Polygon <: Drawable
     vertices::Vector{Point}
 end
 
+"""
+    rectangle = RectanglePoints(p1,p2)
+
+A `Drawable` rectangle i.e. a closed path where parameters `p1` and `p2` 
+are diagonally opposite vertices of the rectangle.
+"""
```

Comment:
Would also be nice to have some examples here.

```suggestion
"""
    rectangle = RectanglePoints(p1, p2)
    rectangle = RectanglePoints(x1, y1, x2, y2)

A `Drawable` rectangle i.e. a closed path where parameters `p1` and `p2` 
are diagonally opposite vertices of the rectangle.
"""
```